src/gl_limits.rs

```diff
@@ -4,12 +4,16 @@ use gleam::gl;
 #[cfg_attr(feature="serde_serialization", derive(Serialize, Deserialize))]
 pub struct GLLimits {
     pub max_vertex_attribs: u32,
+    pub max_tex_size: i32,
```

Comment:
Arrgh, missed this previously, sorry! This should be unsigned (texture sizes are guaranteed to be positive).